examples/jsm/nodes/functions/PhysicalLightingModel.js

```diff
@@ -113,6 +212,38 @@ class PhysicalLightingModel extends LightingModel {
 
 		}
 
+		if ( this.iridescence === true ) {
+
+			const dotNVi = transformedNormalView.dot( positionViewDirection ).clamp();
+
+			this.iridescenceFresnel = evalIridescence( float( 1.0 ), iridescenceIOR, dotNVi, iridescenceThickness, specularColor );
+			this.iridescenceF0 = Schlick_to_F0( { f: this.iridescenceFresnel, f90: 1.0, dotVH: dotNVi } );
+
+		}
+
+	}
+
+	// Fdez-Agüera's "Multiple-Scattering Microfacet Model for Real-Time Image Based Lighting"
+	// Approximates multiscattering in order to preserve energy.
+	// http://www.jcgt.org/published/0008/01/03/
+
+	computeMultiscattering( singleScatter, multiScatter, specularF90 = float( 1 ) ) {
+
+		const fab = DFGApprox( { roughness } );
+
+		const Fr = this.iridescenceF0 ? iridescence.mix( specularColor, this.iridescenceF0 ) : specularColor;
+
+		const FssEss = Fr.mul( fab.x ).add( specularF90.mul( fab.y ) );
+
+		const Ess = fab.x.add( fab.y );
+		const Ems = Ess.oneMinus();
+
+		const Favg = specularColor.add( specularColor.oneMinus().mul( 0.047619 ) ); // 1/21
```

Comment:
Can't we just write this constant as 1/21 then? It will be converted into its decimal form in any case while building.